modules/mono/editor/bindings_generator.cpp

```diff
@@ -390,11 +399,23 @@ String BindingsGenerator::bbcode_to_xml(const String &p_bbcode, const TypeInterf
 			pos = brk_end + 1;
 			tag_stack.push_front(tag);
 		} else if (tag == "code") {
-			xml_output.append("<c>");
+			int end = bbcode.find("[", brk_end);
+			if (end == -1) {
+				end = bbcode.length();
+			}
+			String code = bbcode.substr(brk_end + 1, end - brk_end - 1);
+			if (is_csharp_keyword(code)) {
```

Comment:
Since this comes from a `[code]` tag in the XML documentation which generally uses GDScript syntax, it seems to me that if the text happens to be the name of a C# keyword it would be purely coincidental. Therefore, I'm not sure we should make this change.

Reply:
The main words I had in mind for this were `true, false, null`, all of which seemed pretty applicable. Though I never did a full sweep to see what all would get caught in it, and after doing so the results were:
```yaml
1: base
1: class
1: default
1: for
1: is
1: typeof
2: if
2: in
2: new
2: object
2: return
2: string
2: uint
2: using
3: int
4: bool
6: event
8: float
8: internal
112: null
352: false
1562: true
```
So a fair bit more got in the crossfire than I expected. In this case, I could instead have an internal vector of strings to match against instead, similar to [`ignored_types`](https://github.com/godotengine/godot/blob/33188453277601e3f6d1bd24262b67e11a5d7019/modules/mono/editor/bindings_generator.cpp#L119C1)